strings/remove duplicates.dart

```diff
@@ -0,0 +1,28 @@
+// function to remove duplicate in string
```

Comment:
You could add some tests
```suggestion
import 'package:test/test.dart';

// function to remove duplicate in string
```